### PR TITLE
hysteria: 2.4.0, new

### PR DESCRIPTION
**Pull request by Nyovelt**
Topic Description
-----------------

- hyseria: new, 2.4.0

Package(s) Affected
-------------------

- hysteria: 2.4.0-1

Security Update?
----------------

No

Build Order
-----------

```
#buildit hysteria
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`

### CODE REVIEW

File: app-network/hysteria/autobuild/defines [hunk 1]
```diff
@@ -0,0 +1,7 @@
+PKGNAME=hysteria
+PKGSEC=net
+PKGDES="a network proxy software."
```
Comment (MingcongBai) [inline app-network/hysteria/autobuild/defines]: A multi-protocol network proxy

File: app-network/hysteria/autobuild/build [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+abinfo "Installing Hysteria ..."
+install -Dvm755 "$SRCDIR"/hysteria \
+    -t "$PKGDIR"/usr/bin/
+
+abinfo "Installing license (required for MIT) ..."
+install -Dm644 "$SRCDIR"/LICENSE.md \
+    "$PKGDIR"/usr/share/licenses/hysteria/LICENSE.md
+
+abinfo "Installing config directory ..." 
+install -dm755 "$PKGDIR"/etc/hysteria/
```
Comment (MingcongBai) [inline app-network/hysteria/autobuild/build]: Verbosity here.

File: app-network/hysteria/autobuild/build [hunk 3]
```diff
@@ -0,0 +1,13 @@
+abinfo "Building Hysteria ..."
+go build -o hysteria ./app
+
+abinfo "Installing Hysteria ..."
+install -Dvm755 "$SRCDIR"/hysteria \
+    -t "$PKGDIR"/usr/bin/
+
+abinfo "Installing license (required for MIT) ..."
+install -Dm644 "$SRCDIR"/LICENSE.md \
```
Comment (MingcongBai) [inline app-network/hysteria/autobuild/build]: This is unnecessary, Autobuild will handle this.

File: app-network/hysteria/spec [hunk 4]
```diff
@@ -0,0 +1,5 @@
+VER=2.4.0
+REL=1
```
Comment (MingcongBai) [inline app-network/hysteria/spec]: We start at 0, please remove this line.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Comment by jiegec** [comment]
Ping @MingcongBai

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**